pkg/band/band.go

```diff
@@ -138,7 +138,7 @@ func (b Band) FindUplinkDataRate(dr ttnpb.DataRate) (ttnpb.DataRateIndex, DataRa
 // FindDownlinkDataRate returns the downlink data rate with index by API data rate, if any.
 func (b Band) FindDownlinkDataRate(dr ttnpb.DataRate) (ttnpb.DataRateIndex, DataRate, bool) {
 	// NOTE: See notes in FindUplinkDataRate explaining the order of scanning data rates.
-	for i := ttnpb.DataRateIndex(len(b.DataRates) - 1); int(i) >= 0; i-- {
+	for i := ttnpb.DataRateIndex(15); int(i) >= 0; i-- {
```

Comment:
```suggestion
	for i := ttnpb.DATA_RATE_15; i >= ttnpb.DATA_RATE_0; i-- {
```

Reply:
Also the same for `FindUplinkDataRate` since that one falls into the same trap as well.